README.adoc

```diff
@@ -274,11 +323,34 @@ The cached system properties contain only the OS name and user name key and valu
 To see that the `system` service is down, point your browser to the http://localhost:9080/system/properties[http://localhost:9080/system/properties^] URL again.
 You see that the service displays a 503 HTTP response code.
 
-When you're done, change the [hotspot=2]`io_openliberty_guides_system_inMaintenance`
-property value back to `false` in the [hotspot file=0]`resources/CustomConfigSource.json` file.
+Point your browser to https://localhost:9443/metrics/application[https://localhost:9443/metrics/application^] again.
+See the following sample outputs for the `@Fallback` annotated method and the fallback method after a fallback occurs:
```

Comment:
Change to:
"Go to the https://localhost:9443/metrics/application[https://localhost:9443/metrics/application^] URL again. The following example shows sample outputs for the `@Fallback` annotated method and the fallback method after a fallback occurs:"

Reply:
Updated